server/base/src/main/java/org/apache/accumulo/server/util/ECAdmin.java

```diff
@@ -156,55 +162,97 @@ private void listCompactorsByQueue(ServerContext context) {
     if (compactors.isEmpty()) {
       System.out.println("No Compactors found.");
     } else {
-      Map<String,List<ServerId>> m = new TreeMap<>();
+      Map<String, List<ServerId>> m = new TreeMap<>();
       compactors.forEach(csi -> {
         m.putIfAbsent(csi.getResourceGroup(), new ArrayList<>()).add(csi);
       });
       m.forEach((q, c) -> System.out.println(q + ": " + c));
     }
   }
 
-  private void runningCompactions(ServerContext context, boolean details) {
-    CompactionCoordinatorService.Client coordinatorClient = null;
-    TExternalCompactionList running;
-    try {
-      coordinatorClient = getCoordinatorClient(context);
-      running = coordinatorClient.getRunningCompactions(TraceUtil.traceInfo(), context.rpcCreds());
-      if (running == null) {
-        System.out.println("No running compactions found.");
-        return;
-      }
-      var ecidMap = running.getCompactions();
-      if (ecidMap == null) {
-        System.out.println("No running compactions found.");
-        return;
+  private void runningCompactions(ServerContext context, boolean details, String format) {
+  CompactionCoordinatorService.Client coordinatorClient = null;
+  TExternalCompactionList running;
+  
+  try {
+    coordinatorClient = getCoordinatorClient(context);
+    running = coordinatorClient.getRunningCompactions(TraceUtil.traceInfo(), context.rpcCreds());
+
+    if (running == null || running.getCompactions() == null) {
```

Comment:
You could replace the second null check with `getCompactionsSize` instead of returning the map.
```suggestion
    if (running == null || running.getCompactionsSize() == 0) {
```